tasks/telemetry.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Set base path for fluentd certificates
   set_fact:
-    telemetry_certificate_path: "/tmp/hub/fluentd"
+    telemetry_certificate_path: "/etc/hub/fluentd"
```

Comment:
`/etc/hub/fluentd/ssl` would be even more findable but this is good too!

Reply:
updated path accordingly.